src/include/OpenImageIO/deepdata.h

```diff
@@ -60,17 +62,21 @@ class OIIO_API DeepData {
     /// Copy assignment
     const DeepData& operator=(const DeepData& d);
 
-    /// Clear the vectors and reset size to 0.
+    /// Reset the `DeepData` to be equivalent to its empty initial state.
     void clear();
-    /// Deallocate all space in the vectors
+    /// In addition to performing the tasks of `clear()`, also ensure that
+    /// all allocated memory has been truly freed.
     void free();
 
-    /// Initialize size and allocate nsamples, pointers.
+    /// Initialize the `DeepData` with git specified number of pixels,
```

Comment:
typo: `git` ?

Reply:
um... must have thought the focus was on a terminal window...